portfolio/src/main/webapp/calendar.js

```diff
@@ -0,0 +1,50 @@
+class CalendarEvent{
+  constructor(name, startTime, endTime) {
+    this.name = name;
+    this.startTime = startTime;
+    this.endTime = endTime;
+  }
+}
+
+/** 
+ * Onclick function for "Add this event" button.
+ * Retrieves the new event's name, start and end times, and creates a new 
+ * CalendarEvent from this information. Adds the new event to the 
+ * event list. 
+ */
+function createNewCalendarEvent() {
+  const startTime = document.getElementById('new-event-start-time').value;
```

Comment:
1. Let's try to indicate time unit in the variable names.
2. Let's try to use Date objects wherever possible.

Reply:
Updated to date object.